internal/polling/polling.go

```diff
@@ -0,0 +1,86 @@
+package polling
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log/slog"
+	"net/http"
+	"regexp"
+	"strconv"
+
+	"sync/atomic"
+
+	"gophermart/internal/model"
+
+	"github.com/go-resty/resty/v2"
+)
+
+var Downtime atomic.Uint64
+var RPM atomic.Uint64
+
+type accrualResp struct {
+	Order   int      `json:"order,string"`
+	Status  string   `json:"status"`
+	Accrual *float64 `json:"accrual,omitempty"`
+}
+
+type Store interface {
```

Comment:
Здесь можно добавить аннотацию для генерации моков.  В этом случае можно генерировать моки при помощи команды `go generate ./...`.

```suggestion
//go:generate mockgen -destination ./internal/polling/store_mock.go -package polling gophermart/internal/polling Store
type Store interface {
```